docs/MO_DG/prepare_model/convert_model/Converting_Model.md

```diff
@@ -1,32 +1,52 @@
-Resulting Intermediate Representation will not be resizable with the help of OpenVINO Runtime API.
+```sh
+mo --input_model ocr.onnx --input data,seq_len --input_shape [-1,150,200,1],[-1]
+```
 
-## Parameters for Pre-Processing
-Input data may require pre-processing such as `RGB<->BGR` conversion and mean and scale normalization. To learn about Model Optimizer parameters used for pre-processing, refer to [Optimize Preprocessing Computation](../Additional_Optimizations.md).
+## When to Specify `--static_shape` Command Line Parameter
+If the `--static_shape` command line parameter is specified, the Model Optimizer evaluates shapes of all operations in the model (shape propagation)
+for a fixed input(s) shape(s). During the shape propagation the Model Optimizer evaluates operations *Shape* and removes them from the computation graph.
+With that approach, the initial model which can consume inputs of different shapes may be converted to IR working with the input of one fixed shape only.
+For example, consider the case when some blob is reshaped from 4D of a shape *[N, C, H, W]* to a shape *[N, C, H \* W]*.
+During the model conversion the Model Optimize calculates output shape as a constant 1D blob with values *[N, C, H \* W]*.
+So if the input shape changes to some other value *[N, C, H1, W1]* (it is possible scenario for a fully convolutional model) then the reshape layer
+becomes invalid. Resulting Intermediate Representation will not be resizable with the help of OpenVINO Runtime API.
 
 ## See Also
-* [Configuring the Model Optimizer](../../Deep_Learning_Model_Optimizer_DevGuide.md)
-* [Model Cutting](Cutting_Model.md)
-* [Optimize Preprocessing Computation](../Additional_Optimizations.md)
-* [Convert TensorFlow Models](Convert_Model_From_TensorFlow.md)
-* [Convert ONNX Models](Convert_Model_From_ONNX.md)
-* [Convert PyTorch Models](Convert_Model_From_PyTorch.md)
-* [Convert PaddlePaddle Models](Convert_Model_From_Paddle.md)
-* [Convert MXNet Models](Convert_Model_From_MxNet.md)
-* [Convert Caffe Models](Convert_Model_From_Caffe.md)
-* [Convert Kaldi Models](Convert_Model_From_Kaldi.md)
\ No newline at end of file
+* [Introduction](../../Deep_Learning_Model_Optimizer_DevGuide.md)
+* [Cutting Off Parts of a Model](Cutting_Model.md)
+* [Optimizing Preprocessing Computation](../Additional_Optimizations.md)
+* [Compression of a Model to FP16](../FP16_Compression.md)
+* [Converting TensorFlow Models](Convert_Model_From_TensorFlow.md)
+* [Converting ONNX Models](Convert_Model_From_ONNX.md)
+* [Converting PyTorch Models](Convert_Model_From_PyTorch.md)
+* [Converting PaddlePaddle Models](Convert_Model_From_Paddle.md)
+* [Converting MXNet Models](Convert_Model_From_MxNet.md)
+* [Converting Caffe Models](Convert_Model_From_Caffe.md)
+* [Converting Kaldi Models](Convert_Model_From_Kaldi.md)
```

Comment:
do we need so long list? I suppose the page is renamed from conversion steps to input shapes steps, so `See also` can refer to `Changing input shapes` in runtime `openvino_docs_IE_DG_ShapeInference.html`

Reply:
Corrected "See also" list. Now we can't easily change these filenames because URLs will be broken. We need to make aliases probably for existing names. @andrew-zaytsev, please advise - can we rename doc files?